client/src/Components/Post/Postdetails.js

```diff
@@ -0,0 +1,35 @@
+import './postpage.css';
+import heart from './heart.png'
+import share from './share.png'
+
+function Postdetails(props){
+    const likesbutton= async (e)=>{
+        const data = new FormData();
+        data.append("id",props.id)
+        console.log('like',props.id)
+        fetch('http://localhost:9000/post/' + props.id, {
+            method: 'PUT',
+            body: data,
+            
+        }).then(res => res.json())
+        .then((data)=>console.log(data));
+        window.location.href=".";
```

Comment:
dont use window.location to redirect, instead use this.props.history to redirect to post page